freqtrade/optimize/backtesting.py

```diff
@@ -383,14 +383,17 @@ def _get_close_rate(self, sell_row: Tuple, trade: LocalTrade, sell: SellCheckTup
 
     def _get_adjust_trade_entry_for_candle(self, trade: LocalTrade, row: Tuple
                                            ) -> LocalTrade:
+        current_entry_rate = current_exit_rate = row[OPEN_IDX]
+        current_rate = current_entry_rate
```

Comment:
There's little point in this assignment.
Best assign to one variable (probably `current_rate`) - and use that throughout the method.

using 3 names for the same thing will not improve clarity (the only reason to do this) - but actually reduce it - as it's unclear at first glance at the wrapper lines that we're passing in the same value 3 times.

Reply:
updated